docs/app/_components/CommandItems.tsx

```diff
@@ -255,3 +255,33 @@ export const PagesItem = ({
     </CommandGroup>
   );
 };
+
+export const InternalPage = ({ classNames, setOpen }: ChangeOpenItemProps) => {
+  const router = useRouter();
+  const params = useSearchParams();
+
+  const goto = ({ slug, hash = '' }: { slug: string; hash?: string }) => {
```

Comment:
Can't we reuse the goto function from PagesItem? Only `setPages([]);` is missing. Don't we need this?

Reply:
we don't need setPages because its for the subitems

Reply:
but I will have a look again and see if I can reuse it :)

Reply:
can you have a look again please? :) @aromko